adaboost/core/operations_impl.cpp

```diff
@@ -4,11 +4,36 @@
 #include<adaboost/utils/utils.hpp>
 #include<adaboost/core/operations.hpp>
 #include<iostream>
+#include<cmath>
 
 namespace adaboost
 {
     namespace core
     {
+
+        template <class data_type_vector>
+        void fill(const data_type_vector value, const Vector<data_type_vector>&vec)
+        {
+            for(unsigned i = 0; i < vec.get_size(); i++)
+            {
+                vec.get_data_pointer()[i] = value;
+            }
+        }
+
+
+        template <class data_type_matrix>
+        void fill(data_type_matrix value, const Matrix<data_type_matrix>&mat)
+        {
+            for(unsigned i = 0; i < mat.get_rows(); i++)
+            {
+                for(unsigned j = 0; j < mat.get_cols(); j++)
+                {
+                    mat.get_data_pointer()[i*mat.get_cols() + j] = value;
+                }
+            }
```

Comment:
```suggestion
            data_type_mat* matPtr = mat.get_data_pointer();
            std::fill(matPtr, matPtr + mat.get_rows()*mat_get_cols(), value);
```
Works?

Reply:
This is giving some small syntactical errors, I will commit this suggestion and then again push my commit with debugged update.